invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/index.js

```diff
@@ -25,49 +26,47 @@ const getConfigFromDataAttribute = (element, attr) => {
   return JSON.parse(dataValue);
 };
 
-const TAB_PANES = [
-  {
-    configDataAttribute: "invenio-search-user-uploads-config",
-    label: i18next.t("Uploads"),
-    pathname: "uploads",
-  },
-  {
-    configDataAttribute: "invenio-search-user-communities-config",
-    label: i18next.t("Communities"),
-    pathname: "communities",
-  },
-  {
-    configDataAttribute: "invenio-search-user-requests-config",
-    label: i18next.t("Requests"),
-    pathname: "requests",
-  },
-];
-
 const replaceURLPathname = (newPathname) =>
   window.history.replaceState({}, "", `/me/${newPathname}`);
 
-class DashboardTabs extends Component {
+class DashboardMenu extends Component {
   constructor(props) {
     super(props);
-    const activeTabName = getConfigFromDataAttribute(
-      rootElement,
-      _camelCase("active-tab-name")
-    );
-    const communitiesEnabled = getConfigFromDataAttribute(
+    const activeMenuName = getConfigFromDataAttribute(
       rootElement,
-      _camelCase("communities-enabled")
+      _camelCase("active-menu-element")
     );
 
-    this.ACTIVE_TAB_PANES = communitiesEnabled ? TAB_PANES : [TAB_PANES[0]];
-
-    const routes = this.ACTIVE_TAB_PANES.map((pane) => pane.pathname);
+    this.menuElements = [
```

Comment:
You could have kept it outside the component, so that it was a static obj.
Why having a list and not an obj? You would avoid the `find`:
```
MENU_ITEMS = {
    "uploads": {...},
    ...
}
``` 
and then:
```
handleItemClick = (e, { name }) => {
    const activeMenuElement = MENU_ITEMS[_lowerCase(name)].key
    this.setState({
      currentActiveMenuElement: activeMenuElement,
    });
    replaceURLPathname(_lowerCase(name));
  };
```